db/model/Gdoc/htmlToEnriched.ts

```diff
@@ -732,13 +732,29 @@ function cheerioToArchieML(
         unwrapElement(element, context)
 
     const span = cheerioToSpan(element)
-    if (span)
+    if (span) {
+        // Special handling for a unique span that we use to mark the first published date
+        if (
+            span.spanType === "span-simple-text" &&
+            span.text.trim().match(/published/)
```

Comment:
This is probably too generous now and I should make sure it only applies to the first span in the article, not any span that contains the word "published"